mocks/stats-list.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": "lwkaFmtuCoeKSFhbndTC",
+    "tag": "my-custom-tag",
```

Comment:
We said today or yesterday that we should probably remove `tag` since we are not using it for the moment (and the user cannot set it anyway)!
I have done a PR#25 for server for this today.

If you agree, it will be necessary to remove it from here too!